### PR TITLE
Improve the CPFP output detection

### PR DESCRIPTION
**Pull request by darosior**
So in the end i don't think adding a proprietary field is worth it.

### CODE REVIEW

File: src/transactions/tests_helpers.rs [hunk 1]
```diff
@@ -209,6 +209,7 @@ macro_rules! roundtrip {
         }
 
         let serialized_tx = $tx.to_string();
+        eprintln!("{}", serialized_tx);
```
Comment (darosior) [inline src/transactions/tests_helpers.rs:212]: Woops

File: src/error.rs [hunk 2]
```diff
@@ -233,6 +234,11 @@ impl fmt::Display for PsbtValidationError {
                 "This input refers to an output of invalid type: '{:#?}'",
                 i
             ),
+            Self::InvalidCountOuputWithDerivations(c) => write!(
+                f,
+                "An incorrect number of PSBT outputs had the bip32 derivation set. Expected {}.",
```
Comment (danielabrozzoni) [inline src/error.rs:239]: giant nit, you could say how many were provided

Comment (darosior) [reply src/error.rs:239]: Meh :(

Comment (darosior) [reply src/error.rs:239]: Actually if we do that we should as well put the min.. And that becomes dirty. Instead i just replaced expected by the actual invalid count.

**Comment by danielabrozzoni** [comment]
ACK 8434188d6f6289d3f59c1c8aa22f123418ac4614

**Pull request merged**